static/js/flow/discussionTypes.js

```diff
@@ -47,7 +45,6 @@ export type ChannelForm = {
 
 export type ChannelAppearanceEditValidation = {
   title:              string,
-  description:        string,
```

Comment:
kind of odd that we even had this field here, since the validator didn't have a check for `description`, but oh well!